app/javascript/App.js

```diff
@@ -0,0 +1,21 @@
+import { createBrowserRouter, RouterProvider} from "react-router-dom";
+import React from 'react';
+
+import Greetings from './Components/Greetings';
+
+const router = createBrowserRouter([
+  {
+    path: "/greetings",
+    element: <Greetings />,
+  },
+]);
+
+function App() {
+  return (
+    <React.StrictMode>
+      <RouterProvider router={router} />
+    </React.StrictMode>
+  );
+}
```

Comment:
- [ ] I still do not see anything on the screen due to perhaps routing issues 👇🏾 

![image](https://user-images.githubusercontent.com/84156714/220296846-8c02d6c8-b048-491a-88b4-7f7fcdefb3e0.png)

![image](https://user-images.githubusercontent.com/84156714/220297255-9abc8a1a-f0bc-4bd0-b074-cdf59243e8c5.png)

Kindly check and be sure it works as expected.

Reply:
I am confident I have set things up right. Created a separate route for greetings different from messages. I can view all messages, create, delete, and update, although I do not think this is a requirement for the project. My app is working as expected and I will need you to check your setups, database, etc as my environments may differ from yours. Attached are screenshots of the app working as expected.
I can clearly view all messages
I can also view a random message using /greeting route 
![003](https://user-images.githubusercontent.com/75745927/220314132-c6e0caaf-168a-47dc-b034-2c60e19f82ea.PNG)
![001](https://user-images.githubusercontent.com/75745927/220314137-8528e7ba-e532-4cea-a91c-574070b9583d.PNG)
![002](https://user-images.githubusercontent.com/75745927/220314140-f71abb74-2c41-4b77-a3c5-b2442a3e0647.PNG)

Reply:
If you still have challenges getting this to run then it would be best to schedule a call so we go through as I again cannot see any errors in my configurations and routes.

Reply:
![004](https://user-images.githubusercontent.com/75745927/220315633-51aae1b9-ab2d-4a3b-8ed8-9d0a092cce44.PNG)

Reply:
@RayhanTabase , I see your screenshots. I was able to set up the project on my machine without issues. But the route `/greetings` was blank. I will suggest you include any setup detail that will help duplicate what you have on your machine. 

Submit the project for review, and while submitting, include a message asking the reviewer for a call session so you both run through the project together.